lecture-4-equivalence/solution-4.tex

```diff
@@ -83,10 +83,10 @@ \section{}
 \begin{enumerate}[(a)]
 \item 34,159 is not a prime number.
 \item Roses are not red and violets are not blue.
```

Comment:
@dantevangelista

Should this be `Roses are not red or violets are not blue.`? (changed `and` to `or`)

Reply:
yes